ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ChatText.qml

```diff
@@ -44,8 +44,10 @@ StyledTextEdit {
                     `p {`+
                         `white-space: pre-wrap;`+
                     `}`+
+                    `a {`+
+                        `color: ${isCurrentUser ? Style.current.white : Style.current.textColor};`+
+                    `}`+
                     `a.mention {`+
-                        `color: ${isCurrentUser ? Style.current.black : Style.current.white};`+
```

Comment:
You should probably keep that line and put `color: Style.current.black ` above. The reason is that the color change is only for mentions